internal/helm/helmchart_structure_exists_test.go

```diff
@@ -85,3 +89,28 @@ func TestShouldPassIfHelmStructureExist(t *testing.T) {
 		t.Errorf("Helm structure exists hence test should pass.")
 	}
 }
+
+func TestShouldPassIfChartYamlIsValid(t *testing.T) {
```

Comment:
Same thing as mentioned with the tests before. Try to name after requirements and not technical details like return types or boolean values

Reply:
Changed to TestShouldPassIfConfigurationSettingsAreCorrect.